recording/test/support/mock_track.ex

```diff
@@ -0,0 +1,24 @@
+defmodule MockTrack do
+  @moduledoc false
+
+  alias ExSDP.Attribute.FMTP
+  alias Membrane.RTC.Engine.Track
+
+  @spec create_track(:audio | :video) :: Track.t()
+  def create_track(type) do
+    %Track{
+      type: type,
+      stream_id: "stream_id",
+      id: "id",
+      origin: "origin",
+      fmtp: %FMTP{pt: nil},
+      encoding: :h264,
+      variants: [:high],
+      clock_rate: 90_000,
+      active?: true,
+      metadata: %{},
+      ctx: %{},
+      framerate: nil
+    }
```

Comment:
I am unsure if providing a Track with type `:audio` and `:encoding` `:h264` is the best approach, but we can leave it for now.

Reply:
it's just to mock track for reports all the data is made up.